rockcraft.yaml

```diff
@@ -0,0 +1,27 @@
+name: kratos
+base: bare
+build-base: ubuntu:22.04
+version: "0.13.0"
+summary: Ory Kratos identity server
+description: |
+  Ory Kratos is a Identity, User Management and Authentication system for the Cloud.
+license: Apache-2.0
+platforms:
+  amd64:
+
+services:
+  kratos:
+    override: replace
+    command: kratos serve -c /etc/config/kratos/kratos.yml --dev --watch-courier
+    startup: enabled
+
+parts:
+  kratos:
+    plugin: go
+    build-snaps:
+      - go/1.19/stable
```

Comment:
Is there a reason we point to go 1.19 instead of 1.20

Reply:
Upstream uses go 1.19 in their Dockerfile, we could probably use 1.20 despite that.